app/src/main/java/org/jellyfin/mobile/player/PlayerFragment.kt

```diff
@@ -204,12 +210,18 @@ class PlayerFragment : Fragment() {
      * Handle current orientation and update fullscreen state and switcher icon
      */
     private fun updateFullscreenStateAndSwitcher(configuration: Configuration) {
-        if (isLandscape(configuration)) requireActivity().enableFullscreen() else requireActivity().disableFullscreen()
-        val fullscreenDrawable = when {
-            !requireActivity().isFullscreen() -> R.drawable.ic_fullscreen_enter_white_32dp
-            else -> R.drawable.ic_fullscreen_exit_white_32dp
+        with(requireActivity()) {
+            // Do not handle any orientation changes while being in Picture-in-Picture mode
+            if (isInPictureInPictureMode)
+                return
+
+            if (isLandscape(configuration)) enableFullscreen() else disableFullscreen()
+            val fullscreenDrawable = when {
+                !isFullscreen() -> R.drawable.ic_fullscreen_enter_white_32dp
+                else -> R.drawable.ic_fullscreen_exit_white_32dp
+            }
```

Comment:
```suggestion
            val fullscreenDrawable = if (isFullscreen())
                R.drawable.ic_fullscreen_exit_white_32dp
            else
                R.drawable.ic_fullscreen_enter_white_32dp
```

Reply:
I'd personally keep it as is, even though the when isn't necessary here, it looks cleaner and makes it easier to see the two used drawable resources.

Reply:
In that case I'd invert the condition, instead of checking for `!isFullscreen()` use `isFullscreen()`. More readable that way.